vita3k/config/include/config/config.h

```diff
@@ -57,5 +57,5 @@ struct Config {
     bool color_surface_debug = false;
     bool hardware_flip = false;
     bool performance_overlay = false;
-
+    int backend = 0;
```

Comment:
forget, not remove news line here

Reply:
Is this really necessary? Most structs like `HostState` and `GuiState`, etc. don't have new lines are the end of definitions.

Reply:
all struct i have see have this news line hmm

Reply:
Here are my examples:
https://github.com/Vita3K/Vita3K/blob/master/vita3k/host/include/host/state.h#L92
https://github.com/Vita3K/Vita3K/blob/master/vita3k/gui/include/gui/state.h#L130
https://github.com/Vita3K/Vita3K/blob/master/vita3k/kernel/include/kernel/state.h#L201
etc.

I think this newline was a mistake?